CMakeLists.txt

```diff
@@ -115,11 +115,17 @@ target_link_libraries(ed ed_core ed_io ed_server)
 #                                               PLUGINS
 # ------------------------------------------------------------------------------------------------
 
+find_package(tf2 REQUIRED)
```

Comment:
Why do the `find_package` here and not including them at the top? Does this speed up stuff?

Reply:
I don't know how catkin works under the hood. But only this plugin uses TF2, so I though, not linking the other libraries against it might be faster.

Reply:
Never really looked into this, but AFAIK this is used to generate a makefile. Hence, all `find_package` calls are executed in this pre-processing step.

Reply:
Yes, but tha linking happens in the makefile. And I think reducing the linking reduces the build time.